src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -263,16 +265,20 @@ export class HuiCardPicker extends LitElement {
   }
 
   private _loadCards() {
-    let cards: Card[] = coreCards.map((card: Card) => ({
-      name: this.hass!.localize(
-        `ui.panel.lovelace.editor.card.${card.type}.name`
-      ),
-      description: this.hass!.localize(
-        `ui.panel.lovelace.editor.card.${card.type}.description`
-      ),
-      isSuggested: this.suggestedCards?.includes(card.type) || false,
-      ...card,
-    }));
+    let cards: Card[] = coreCards
+      .filter((card: Card) =>
+        this.isSectionsView ? true : !card.sectionsViewOnly
+      )
```

Comment:
Can we instead just skip the filter? It will still iterate over all the items now

```ts
const viewCards = this.isSectionsView ? coreCards : coreCards.filter(card => card.sectionsViewOnly);
let cards = viewCards.map(...)
```

Reply:
Why is it only for sections view anyway? The use case is to insert it into a grid card or stack card which are used in all view types.

Reply:
In section view we have the layout options for the card to set it to the size as you wish.
But I can look into it to set it to a default size when in grid or stack card.

Reply:
No default size pls. It shouldn't have a size. When you put it in a grid, the grid determines the size. People are using an empty grid or horizontal-stack for this now. They don't want the card to have a size in many cases

Reply:
See https://github.com/home-assistant/frontend/issues/24369 and the discussion linked in there